WordPress/gutenberg
# Block Theme Previews: Remove "Looking for Template Parts?" hint.

## What?

Remove "Looking for Template Parts?" hint.

Fixes https://github.com/WordPress/gutenberg/issues/56668

## Why?


The hint has been there for two releases and it might be confusing for first-time users.

## How?

Removing the template part hint file and call.

## Testing Instructions




1. Go to Appearance > Themes
2. Preview a theme (TT3)
3. Confirm that the hint is not there.

## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/252415/6b955032-000e-47e3-90d6-2a62773b8848
